cmake/api_macros.cmake

```diff
@@ -65,6 +65,11 @@ macro(OPENDDS_GET_SOURCES_AND_OPTIONS
   endforeach()
 endmacro()
 
+macro(OPENDDS_NO_SUPPRESS_ANYS)
+  list(REMOVE_ITEM OPENDDS_DDS_BASE_IDL_FLAGS -Sa -St)
+  list(REMOVE_ITEM OPENDDS_TAO_BASE_IDL_FLAGS -Sa -St)
```

Comment:
Would this need to split the two args into two different CMake commands to handle `-St -Sa` (or if they appear with other args mixed in)?

Reply:
I think it handles the values as different entities.

The documentation says: list(REMOVE_ITEM <list> <value> [<value> ...])

Which indicates each value is probably actually done as a seperate call of remove item.